src/stores/concatStore.ts

```
@@ -0,0 +1,5 @@
+import { create } from 'zustand';
+
+interface ConcatState {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)